### PR TITLE
Coop copy: companies, locations

### PR DESCRIPTION
**Pull request by RobbieZhuang**
Added some copy and fixed some graphs:

Screen Shot 2023-04-16 at 11 44 08 AM
Screen Shot 2023-04-16 at 11 44 14 AM

### CODE REVIEW

File: content/coop.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <h1 id="coop">Co-op</h1>
 <h3>Companies</h3>
 <p class="full">
-  <span class="big"> At least 150 companies have hired an SE student. </span>
+  <span class="big"> At least 166 companies have hired an SE student. </span>
   Students had the opportunity to work in many different types of environments,
-  including working at a 5 person startup, writing code in Shanghai, researching
-  with professors, and working at a big tech company. While a majority of
+  including lean startups, writing code in Japan, researching
+  with professors, and working at at large tech companies. While a majority of
```
Comment (wesley-a-leung) [inline content/coop.html]: Typo: `at at` -> `at`

File: content/coop.html [hunk 2]
```diff
@@ -37,24 +37,14 @@ <h3>Companies</h3>
 <h3>Locations</h3>
 <p class="full">
   <span class="big">
-    77% of respondents co-oped in at least 2 different countries.
+    Coops occured all across North America
```
Comment (wesley-a-leung) [inline content/coop.html]: `Co-ops` instead of `Coops`?

**Review by wesley-a-leung — APPROVED** [review]
Few typos, but otherwise fine.

**Comment by RobbieZhuang** [comment]
ty for the review @wesley-a-leung!

**Pull request merged**